decred/decred/util/helpers.py

```diff
@@ -156,7 +156,7 @@ def readINI(path, keys):
     Returns:
         dict: Discovered keys and values.
     """
-    config = configparser.ConfigParser()
+    config = configparser.ConfigParser(strict=False)
```

Comment:
What's the reasoning here? I'd rather see an error on duplicates.

Reply:
Thanks for taking a look. The `rpclisten` parameter can be specified multiple times in the [dcrd config file](https://github.com/decred/dcrd/blob/b60c60ffe98bcea5becc4e9e4d2a4fe6152401f6/sampleconfig/sampleconfig.go#L190-L191) (to listen on multiple interfaces/ports). I added a [test case](https://github.com/decred/tinydecred/pull/191/files#diff-a99f9bbbcd0605f4659bc0e668ddeee4R70) for that scenario as well.